.github/workflows/run-tests.yaml

```diff
@@ -62,7 +62,7 @@ jobs:
       - name: Run black
         run: poetry run black . --check
       - name: Run isort
-        run: poetry run isort . --check-only --profile black
+        run: poetry run isort . --check-only --profile black --line-length=80
```

Comment:
Es wäre besser isort nicht nur in der ci/cd zu konfigurieren sondern generell, so dass andere nicht in daselbe Problem laufen (obwohl es sonst niemand zu haben scheint?). isort sollte über die pyproject.toml oder eigene Konfig-datei konfigurierbar sein kann ich aber auch schauen

Reply:
Gut zu wissen, hatte bisher pre-commit hook und tests angepasst, das hatte für mich bisher alles abgedeckt

Reply:
Getestet und bisher funktionierend in [7af6481](https://github.com/CorrelAid/genesis-python/pull/69/commits/7af64814351dae5b477328d52b3b7567412a0ea8) - resolved